### DISCUSSION TITLE
How can I add packages into Macad.OCCT

### DISCUSSION THREAD

**Discussion by RileyGe** [Macad3D/Macad3D#29] [Q&A] [answered]
If I am wrong, please correct me.

I want add two packages(GC/GCE2d) to the Macad.OCCT. To do this, I go to the project `WrapperGenerator`, find the file `Configuration.cs`, move GC and GCE2d from line 32 to line 34(uncomment the package GC and GCE2d).

![image](https://user-images.githubusercontent.com/26666371/192480707-740cc430-4441-4869-b5b9-48d1ace8f633.png)

Then I start ScriptConsole.cmd and run `occt generate`, everything seems OK.

But after regenerated occt, the GC.cpp and GC.h(the same of GCE2d.cpp and GCE2d.h) not appeared on the Macad.OCCT->Generated->ModelingData->TKGeomBase folder.

![image](https://user-images.githubusercontent.com/26666371/192481210-427be8c4-384e-468b-9f8e-ca655e7fe207.png)

I can build the project successfully, but I can use the two packages.

Is there something I do wrong? How can I add packages into Macad.OCCT?

**Comment by dipts** [comment, marked as answer]
Everything correct so far, you only need to include the new C++ files manually into the project by drag'n'drop.

**Reply by RileyGe** [reply]
Thank you very much. After running `occt generate` command, the files are generated in the `Macad.Occt\Generated`. I dragged the files to the target folder in Visual Studio. After building the project again, everything is OK.

**Comment by wujianming** [comment]
Macad.Occt\Generated done.

what is the next  step?

**Reply by dipts** [reply]
The packages should then be able to be used. What is not working?
